src/posthog-core.ts

```diff
@@ -1113,6 +1168,7 @@ export class PostHog {
     reset(reset_device_id?: boolean): void {
```

Comment:
Should we reset this when the session id resets?

Reply:
I don't think we should. The SessionStorage is basically "as long as the window is open" which is not all that long (unless you are @neilkakkar...)